swagger.yaml

```diff
@@ -0,0 +1,133 @@
+swagger: "2.0"
+info:
+  description: "The Search API is optimized to help you find the specific item you're looking for (e.g., a specific user, a specific file in a repository, etc.). Think of it the way you think of performing a search on Google. It's designed to help you find the one result you're looking for (or maybe the few results you're looking for). Just like searching on Google, you sometimes want to see a few pages of search results so that you can find the item that best meets your needs. To satisfy that need, the GitHub Search API provides up to 1,000 results for each search."
+  version: "1.0.0"
+  title: "Search"
+host: "developer.github.com"
+basePath: "/v3"
+tags:
+- name: "Search repositories"
+  description: "Find repositories via various criteria. This method returns up to 100 results per page."
+schemes:
+- "http"
+paths:
+  /search/repositories:
+    get:
+      tags:
+      - "Search repositories"
+      summary: "get all the film resources"
```

Comment:
Copy + Paste should be done only when you can review all the things that you have copied.